topics/nginx/index.md

```diff
@@ -0,0 +1,13 @@
+---
+created_by: Igor Sysoev
+display_name: nginx
+github_url: https://github.com/nginx/nginx
+logo: nginx.png
+related: nginx-server, nginx-proxy, nginx-cache, nginx-unit, nginx-plus
+released: October 4, 2004
+short_description: nginx is a high performance free open source web server.
+topic: nginx
+url: https://nginx.org/
+wikipedia_url: https://en.wikipedia.org/wiki/Nginx
+---
+nginx is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server. Nginx was written with an explicit goal of provide high performance on sites with hight traffic, so it can be used as a reverse proxy, load balancer and HTTP cache.
```

Comment:
```suggestion
nginx is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server. Nginx was written with an explicit goal of provide high performance on sites with high traffic, so it can be used as a reverse proxy, load balancer and HTTP cache.
```

Reply:
Fixed.